locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -746,20 +746,20 @@ msgstr "Keine Ergebnisse"
 #, javascript-format
 msgid "you have 1 vote left"
 msgid_plural "you have %s votes left"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Sie haben noch %s Stimmen."
+msgstr[1] "Sie haben noch %s Stimmen."
 
 #: meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx:12
 msgid "modified on"
-msgstr "geändert am"
+msgstr "bearbeitet am"
```

Comment:
Which wording do we use in the other (Django) list? I couldn't find "modified on" and I think, we should use the same wording for both lists. I will merge anyway, but could you check that out, please?

Reply:
@sabinammm Did you see my comment? Did you find out about the other wording?